docs/devenv_rhel8.md

```diff
@@ -85,20 +85,33 @@ make srpm
 The artifacts of the build are located in the `_output/rpmbuild` directory.
 ```bash
 $ cd ~/microshift/ && find . -name \*.rpm
-./_output/rpmbuild/RPMS/x86_64/microshift-4.10.0-nightly_1654189204_34_gc871db21.el8.x86_64.rpm
-./_output/rpmbuild/RPMS/x86_64/microshift-4.10.0-networking-nightly_1654189204_34_gc871db21.el8.x86_64.rpm
-./_output/rpmbuild/RPMS/noarch/microshift-selinux-4.10.0-nightly_1654189204_34_gc871db21.el8.noarch.rpm
-./_output/rpmbuild/SRPMS/microshift-4.10.0-nightly_1654189204_34_gc871db21.el8.src.rpm
+./_output/rpmbuild/RPMS/x86_64/microshift-4.12.0-nightly_20220908132255_152_g4267d5d1.el8.x86_64.rpm
+./_output/rpmbuild/RPMS/x86_64/microshift-networking-4.12.0-nightly_20220908132255_152_g4267d5d1.el8.x86_64.rpm
+./_output/rpmbuild/RPMS/noarch/microshift-selinux-4.12.0-nightly_20220908132255_152_g4267d5d1.el8.noarch.rpm
+./_output/rpmbuild/SRPMS/microshift-4.12.0-nightly_20220908132255_152_g4267d5d1.el8.src.rpm
 ```
 
 ## Run MicroShift Executable
 Log into the development virtual machine with the `microshift` user credentials.
 
 ### Runtime Prerequisites
+When working a MicroShift based on a pre-release _minor_ version `Y` of OpenShift, the corresponding PRM repo `rhocp-4.$Y-for-rhel-8-$ARCH-rpms` won't be available yet. In that case, use the `Y-1` version. Then, if you have VPN access to the Red Hat build systems, you can add the corresponding puddle repo.
+
```

Comment:
Let's separate the first statement about _minor_ versions and the puddle repo. The puddle repo-related information can be put in a note including the .repo file configuration.

Otherwise, it's a bit confusing.

Reply:
Updated.